fastlane/en-US/full_description.txt

```diff
@@ -0,0 +1,5 @@
+Gotify is a service for sending and receiving push notifications.
```

Comment:
Could you reuse the images from the play store listing, because there are application images there. I think the images from the play store listing are enough we don't need images for every possible setting, like theme, client name selection, relative/absolute time etc.

Reply:
Where can I find these images?

Reply:
https://play.google.com/store/apps/details?id=com.github.gotify

Reply:
For F-Droid there are two issues with these images. They contain screen decoration (header and footer), which should be cropped out. I can do that. They are also in German, so I can add them to the German part of Fastlane and add keep my screenshots for the English part. Okay? Alternatively, if you send me English version of yours, I can add those to the English part.

Reply:
I'll create the images hopefully later this week and add them directly to this PR.

Reply:
I don't have permission to push on this branch, I'll push directly to master.